packages/instant/src/containers/selected_erc20_asset_amount_input.ts

```diff
@@ -77,6 +80,14 @@ const updateBuyQuoteAsync = async (
     errorUtil.errorFlasher.clearError(dispatch);
     // invalidate the last buy quote.
     dispatch(actions.updateLatestBuyQuote(newBuyQuote));
+
+    // set error if user doesn't have appropriate balance
```

Comment:
we should also either disable the buy button in this case or flash the same message when the user tries to click buy

Reply:
good point -- i think it'd be easier just to flash again if they click buy